internal/proto/controller/api/resources/authmethods/v1/auth_method.proto

```diff
@@ -74,77 +74,77 @@ message AuthMethod {
 message PasswordAuthMethodAttributes {
   // The minimum length allowed for user names for Accounts in this Auth Method.
   uint32 min_login_name_length = 10
-      [json_name = "min_login_name_length", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.min_login_name_length" that: "MinLoginNameLength" }];
+      [json_name = "min_login_name_length", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.min_login_name_length" that: "MinLoginNameLength" }];  // @gotags: `class:"public"`
 
   // The minimum length allowed for passwords for Accounts in this Auth Method.
   uint32 min_password_length = 20
-      [json_name = "min_password_length", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.min_password_length" that: "MinPasswordLength" }];
+      [json_name = "min_password_length", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.min_password_length" that: "MinPasswordLength" }];  // @gotags: `class:"public"`
 }
 
 // The attributes of an OIDC typed auth method.
 message OidcAuthMethodAttributes {
   // Output only. The state of the auth method. Will be "inactive",
   // "active-private", or "active-public".
-  string state = 10 [json_name = "state"];
+  string state = 10 [json_name = "state"];  // @gotags: `class:"sensitive"`
 
   // The issuer URL. Boundary expects only the schema, host, and port and will
   // strip off ".well-known/openid-configuration" if present. This will be
   // used for configuration discovery as well as for validation of the "iss"
   // claim.
-  google.protobuf.StringValue issuer = 20 [json_name = "issuer", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.issuer" that: "Issuer" }];
+  google.protobuf.StringValue issuer = 20 [json_name = "issuer", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.issuer" that: "Issuer" }];  // @gotags: `class:"sensitive"`
 
   // An OAuth 2.0 Client Identifier valid at the Authorization Server.
   google.protobuf.StringValue client_id = 30
-      [json_name = "client_id", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_id" that: "ClientId" }];
+      [json_name = "client_id", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_id" that: "ClientId" }];  // @gotags: `class:"sensitive"`
```

Comment:
this is public and is/can be used in FE on the browser without the client_secret (which is secret)